packages/xo-web/src/common/xo/index.js

```diff
@@ -1010,6 +1020,34 @@ export const startVms = vms =>
       )
     )
 
+    if (forbiddenStart.length === 0 && vmsWithDuplicatedMacAdresses.length === 0) {
+      if (nErrors === 0) {
+        return
+      }
+
+      return error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
+    }
+
+    if (vmsWithDuplicatedMacAdresses.length > 0) {
+      // Retry without checking MAC addresses
+      confirm({
+        title: _('forceStartVm'),
+        body: _('vmsWithDuplicatedMacAddressesMessage', { nVms: vmsWithDuplicatedMacAdresses.length }),
+      }).then(async () => {
+        await Promise.all(
+          map(vmsWithDuplicatedMacAdresses, id =>
+            _call('vm.start', { id, checkMacAddresses: false }).catch(reason => {
+              if (forbiddenOperation.is(reason)) {
+                forbiddenStart.push(id)
+              } else {
+                nErrors++
+              }
+            })
+          )
+        )
+      })
+    }
+
     if (forbiddenStart.length === 0) {
```

Comment:
This case shouldn't be possible, did you forget to remove it?

Reply:
Now, it's possible.